underlay/src/main/resources/config/underlay/pilotsynthea2022q3/underlay.json

```diff
@@ -59,14 +59,25 @@
     "pilotsynthea2022q3/gender",
     "pilotsynthea2022q3/race",
     "pilotsynthea2022q3/yearOfBirth",
+    "pilotsynthea2022q3/measurements",
+    "pilotsynthea2022q3/devices",
+    "pilotsynthea2022q3/visits",
+    "pilotsynthea2022q3/cpt4",
+    "pilotsynthea2022q3/icd9CM",
+    "pilotsynthea2022q3/icd9Proc",
+    "pilotsynthea2022q3/icd10CM",
+    "pilotsynthea2022q3/icd10PCS",
```

Comment:
added this line as well.

Reply:
These lines reference directories under `config/criteria/pilotsynthea2022q3` that you'll need to add. You can look at the `sd` selector configs as a starting point.